generated/nifake/README.rst

```diff
@@ -0,0 +1,21 @@
+This is the MIT license: http://www.opensource.org/licenses/mit-license.php
+
+Copyright (c) 2017, National Instruments Corp.
+NI-DMM, NI-DCPower, NI-FGEN, NI-SCOPE, NI-SWITCH, NI Switch Executive are trademarks of National
+Instruments.
```

Comment:
year is old, list of drivers is old

Is this true elsewhere? May need to open an issue.

Reply:
See #1183